src/System.Memory/tests/Span/SequenceCompareTo.string.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.SpanTests
+{
+    public static partial class SpanTests
+    {
+        [Fact]
+        public static void ZeroLengthSequenceCompareTo_String()
```

Comment:
Can you move these tests (for T = string) into SequenceCompareTo.T.cs? That is where we have other tests for T != primitive numeric types (like byte, int, long).